docker/Dockerfile

```diff
@@ -1,3 +1,17 @@
+FROM node:10 as frontend
+
+WORKDIR "/portal"
+
+# Copy required front-end files only
+# preserve partial checkout structure
+COPY bin/ bin/
```

Comment:
err, this has me scratching my head.  copy `bin/` to `bin/`?  isn't that and the following line a no-op?